tests/test_vfc

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+covered_functions=(
+"clone_all_vfc_repos"
+)
+
+# test_clone_all_vfc_repos() - Verify cloning and pulling source code from repositories
+function test_clone_all_vfc_repos {
+    # TODO(sshank)
+    echo ""
+}
```

Comment:
The functions needs to call the test functions directly like

https://github.com/electrocucaracha/vagrant-onap/blob/master/tests/test_mr#L20-L23